src/filters/string.rs

```diff
@@ -93,6 +94,62 @@ pub fn addslashes(value: Value, _: HashMap<String, Value>) -> TeraResult<Value>
 
 }
 
+/// Capitalizes each word in the string
+pub fn title(value: Value, _: HashMap<String, Value>) -> TeraResult<Value> {
+    let s = try_get_value!("title", "value", String, value);
+    let split_pattern = Regex::new(r"([-\s\(\{\[<]+)").unwrap();
```

Comment:
we should be able to compile that regex into static but that should be ok for now